docs/install/tkgs.rst

```diff
@@ -302,3 +319,16 @@ When you try to create a Notebook Server, you may meet the following error:
     FailedCreate 1s (x2 over 1s) statefulset-controller create Pod test-01-0 in StatefulSet test-01 failed error: pods “test-01-0” is forbidden: PodSecurityPolicy: unable to admit pod: []
 
 This error occurs because Notebook Server creation needs pod creation, and you did not configure the pod security policy correctly. To solve this error, you need to configure pod security policy based on :ref:`configure pod security policy`.
+
+cert-manager webhook is not ready
+---------------------------------
+
+cert-manager is used by Kubeflow components to provide certificates for admission webhooks. When you try to install Kubeflow, you may meet the following error about cert-manager:
+
+.. code-block:: text
+
+    Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp 10.96.202.64:443: connect: connection refused
+
+This is because the webhook is not yet ready to receive request. Wait a couple seconds and retry deploying Kubeflow again.
```

Comment:
This error message indicates that the webhook is not yet ready to receive request. You simply need to wait a couple seconds and retry.
(This is because .... is okay, but may not be formal enough, usually used in conversations.)
(you already have "retry", so do not need "again"